test/e2e/installplan_e2e_test.go

```diff
@@ -2549,3 +2566,123 @@ func TestInstallPlanCRDValidation(t *testing.T) {
 
 	require.Equal(t, v1alpha1.InstallPlanPhaseComplete, fetchedInstallPlan.Status.Phase)
 }
+
+func TestInstallPlanFromBundleImage(t *testing.T) {
+	c := newKubeClient(t)
+	crc := newCRClient(t)
+	catalogSourceName := genName("mock-kiali-")
+
+	// expect to execute in the same namespace OLM is operating in
+	testNamespace := strings.TrimSuffix(testNamespace, "-operator")
+
+	// add RBAC for e2e namespace
+	rbacRole := &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "olm-e2e-configmap-access",
+			Namespace: testNamespace,
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{
+					"",
+				},
+				Verbs: []string{
+					"create", "get", "update",
+				},
+				Resources: []string{
+					"configmaps",
+				},
+			},
+		},
+	}
+	_, err := c.KubernetesInterface().RbacV1().Roles(testNamespace).Create(rbacRole)
+	require.NoError(t, err)
+
+	rbacBinding := &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "olm-e2e-configmap-access-binding",
+			Namespace: testNamespace,
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				APIGroup:  "",
+				Name:      "default",
+				Namespace: testNamespace,
+			},
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "Role",
+			Name:     rbacRole.GetName(),
+		},
+	}
+	_, err = c.KubernetesInterface().RbacV1().RoleBindings(testNamespace).Create(rbacBinding)
```

Comment:
This might actually only be necessary for local dev testing. In a production cluster, OLM should have writes to read configmaps in any namespace, right?